routes/v1.js

```diff
@@ -384,7 +386,7 @@ api.declare({
 
 /** Define tasks */
 api.declare({
-  method:     'post',
+  method:     'put',
```

Comment:
... scheduling a task would then logically be issuing a `PATCH` against `/task/<taskId>` with json object `{"scheduled": true}` ...